_includes/header.html

```diff
@@ -1,6 +1,11 @@
-<header class="site-header">
+<div class="header">
+  <h1><a href="{{ site.baseurl }}/">Overflow</a></h1>
+  <p>the data desk of The Daily Californian</p>
+  <div class="external">
+    <a href="https://facebook.com/dailycal"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+    <a href="https://twitter.com/overflow_dc"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+    <a href="https://dailycal.org"><i class="fa fa-bullseye" aria-hidden="true"></i></a>
+  </div>
```

Comment:
I would also add a little about button somewhere for now, wherever you think looks good, and copy over the description from the current about page.